web/components/Feed/Feed.tsx

```diff
@@ -128,7 +133,7 @@ const filterGroups: FilterGroup[] = [
           { value: Breed.Whippet, label: "Whippet" },
         ],
         dropdown: true,
-        allSelected: true,
+        allSelected: false,
```

Comment:
Gah, after some discussion it seems that we're going with `allSelected: true`. I'm so sorry about the back and forth.